docs/modules/policies/pages/role_policies.adoc

```diff
@@ -0,0 +1,45 @@
+include::ROOT:partial$attributes.adoc[]
+
+= Role policies
+
+include::ROOT:partial$version-check.adoc[]
+
+
+Role policies are simple RBAC policies in which you specify a number of resources; each with a set of permissible actions that the role can carry out on the resource. You can view it as a per-role matrix of resource/action rows/columns whereby each set "cell" is an `ALLOW`. They do not support conditions.
+
+Role policies are evaluated before resource policies but do not guarantee an `ALLOW` -- see below for more details.
+
+[source,yaml,linenums]
+----
+---
+apiVersion: api.cerbos.dev/v1
+rolePolicy:
+  role: "acme_admin" <1>
+  scope: "acme.hr.uk" <2>
+  rules:
+    - resource: leave_request <3>
+      permissibleActions: <4>
+        - view:* <5>
+        - deny
+
+    - resource: salary_record
+      permissibleActions:
+        - edit
+
+    - resource: "*" <6>
+      permissibleActions: ["create"]
+----
+<1> The role to which this policy applies.
+<2> Optional principal xref:scoped_policies.adoc[scope] for this policy.
+<3> The resource to which the following rule applies.
+<4> The list of permissible actions that the role can carry out on the given resource.
+<5> Wildcard actions are supported.
+<6> Wildcard resources are also supported.
+
+== Evaluation
+
+During evaluation of a check request, a role policy is only evaluated if it has a matching principal role and scope. For the policy above, the principal would need to have the role `acme_admin` and the scope `acme.hr.uk`. If a principal has more than one role, the evaluator can return more than one role policy. The union of the `resource` lists is then taken which provides a larger potential pool of matching `resource:action` pairs.
+
+If any role policies are matched according to the above rule, they are evaluated before any resource policies. If a `resource:action` pairing is not explicitly set in any of the role policies, that pairing will immediately resolve to a `DENY`. Otherwise, that given `resource:action` pairing will fall through to any resource policies for further evaluation.
```

Comment:
It just occurred to me that I don't know how the principal policies work at this level. The order should be principal > role > resource, shouldn't it? Is that the case currently?

Reply:
Yeah that's currently the case. Perhaps I should make a note to say "that still happens"

Reply:
Yeah, I think we should say that it goes from most specific to least. Starting with the principal policies, any role policies and, finally, resource policies.